repobee_sanitizer/_sanitize_repo.py

```diff
@@ -25,6 +26,11 @@
 LOGGER = daiquiri.getLogger(__file__)
 
 
+_FilesWithErrors = collections.namedtuple(
+    "_FileWithErrors", "file_rel_path Error"
+)
```

Comment:
* Don't use uppercase for attribute names.
* There can be multiple errors per file, so the attribute should probably be pluralized.
* You should match the variable name with the namedtuple name for clarity.

```suggestion
_FileWithErrors = collections.namedtuple(
    "_FileWithErrors", "file_rel_path errors"
)
```